src/state/application/actions.ts

```diff
@@ -27,3 +27,4 @@ export const setOpenModal = createAction<ApplicationModal | null>('application/s
 export const addPopup =
   createAction<{ key?: string; removeAfterMs?: number | null; content: PopupContent }>('application/addPopup')
 export const removePopup = createAction<{ key: string }>('application/removePopup')
+export const setConnectivityWarning = createAction<{ warn: boolean }>('application/setConnectivityWarning')
```

Comment:
I wonder if you should specify `BlockchainConnectivity` to make it clearer it's not Network connectivity it something else.